package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jellobot",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
typically PRs don’t update the version of things

Reply:
fixed